yfinance_cache/yfc_utils.py

```diff
@@ -63,11 +63,10 @@ def TypeCheckDateEasy(var, varName):
         elif not isinstance(var.tzinfo, ZoneInfo):
             raise TypeError(f"'{varName}' tzinfo must be ZoneInfo not {type(var.tzinfo)}")
 def TypeCheckDateStrict(var, varName):
-    if isinstance(var, pd.Timestamp):
-        # While Pandas missing support for 'zoneinfo' must deny
```

Comment:
We should find out what Pandas version added zoneinfo support and update setup.cfg.template

Reply:
I must've accidentally deleted the `if isinstance(var, pd.Timestamp):` block, my bad